User blocking implementation (#268)
desmos-labs/desmos: docs/developers/msgs/unblock-user.md

> # `MsgBlockUser`

Reviewer: ```suggestion
# `MsgUnblockUser`
```

> This message allows you to block a specific user associated with a given address.

Reviewer: ```suggestion
This message allows you to unblock a previously blocked user.
```

## Structure
```json
{
  "type": "desmos/MsgUnblockUser",
  "value": {

>     "blocker": "<Desmos address that's blocking someone>",

Reviewer: ```suggestion
    "blocker": "<Desmos address of the user that is unblocking another user>",
```

>     "blocked": "<Desmos address that's been blocked>",

Reviewer: ```suggestion
    "blocked": "<Desmos address of the unblocked user>",
```

    "subspace": "<Subspace of the block>"
  }
}   
```

### Attributes
| Attribute | Type | Description |
| :-------: | :----: | :-------- |
| `blocker`  | String | Desmos address of the user that is blocking someone else |

> | `blocked`| String | Desmos address of the blocked user |

Reviewer: ```suggestion
| `blocked`| String | Desmos address of the unblocked user |
```

> | `subspace` | String | Required string that identifies the app |

Reviewer: ```suggestion
| `subspace` | String | String that identifies the app for which the block was valid |
```
